Cargo.toml

```diff
@@ -26,7 +26,6 @@ c2rust-bitfields= "0.3"
 cfg-if = "1.0.0"
 libc= "0.2"
 num_cpus = "1.0"
-paste = "1.0.14"
```

Comment:
I'd prefer not to remove the paste dependency because I'm about to use it to macro-ify the itx fn declarations (there's hundreds so we defs need to use paste to avoid having to list them all out).

Reply:
I can leave it in.

Reply:
Okay, it's added back in now.